### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by chrsbell**
IM SORRY

### CODE REVIEW

File: client/spec/BarGraph.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import StyledBarGraph from '../src/components/BarGraph.jsx';
+import testData from './TestData.js';
+
+describe('<StyledBarGraph />', () => {
+
+  const ratings = testData.ratings;
+
+  it('should render individual bar graphs', () => {
```
Comment (paprikatie) [inline client/spec/BarGraph.spec.jsx:10]: Great idea to test rendering bar graphs!

File: client/src/components/AppModal.jsx [hunk 2]
```diff
@@ -1,74 +1,136 @@
...
+    opacity: 1;
+    transform: none;
+  }
+  100% {
+    opacity: 0;
+    transform: translate(0px, 100%);
+  }
+}
+
+&.enter {
```
Comment (paprikatie) [inline client/src/components/AppModal.jsx:49]: good finds for modal animations! I may try using these instead too

File: client/src/components/BarGraph.jsx [hunk 3]
```diff
@@ -1,58 +1,76 @@
 import React from 'react';
 import styled from 'styled-components';
-import fonts from './Fonts.js';
-import {margins} from './Constants.jsx';
+import {margins, Text, Fonts, animation} from './Constants.jsx';
```
Comment (paprikatie) [inline client/src/components/BarGraph.jsx:3]: Nice code reuse, lots of Constants I  see

File: server/src/index.js [hunk 4]
```diff
@@ -7,11 +7,12 @@ const port = 3000;
...
 app.use(express.static(__dirname + '/../../client/dist'));
 
-// return all stays (remove once endpoints based on stay id implemented)
-app.get('/stays', (req, res) => {
-  db.Room.find({}).exec()
-  .then(docs => {
-    res.status(200).send(docs);
+// return a stay matching the provided roomId
+app.get('/stays/:id', (req, res) => {
+  const roomId = req.params.id.slice(1);
```
Comment (paprikatie) [inline server/src/index.js:12]: use destructuring here -> const { id } = req.params 
*roomID = id? since that's what you named it in your endpoint

**Review by paprikatie — CHANGES_REQUESTED** [review]
I believe Airbnb style guide requires ternaries to be in separate lines like so
boolean === true
  ? <do this>
  : <do this>
and again the destructuring of the server params would show better usage of the endpoints!

File: server/src/index.js (hunk 4 above)
Comment (chrsbell) [reply server/src/index.js:12]: Will do!

File: client/src/components/BarGraph.jsx (hunk 3 above)
Comment (chrsbell) [reply client/src/components/BarGraph.jsx:3]: we out here

**Pull request merged**